frontend/webpack.config.js

```diff
@@ -1,6 +1,11 @@
 // shared config (dev and prod)
 const { resolve } = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
+const dotenv = require("dotenv");
+const webpack = require("webpack");
+
+const myEnv = dotenv.config();
```

Comment:
Looks good 💯

  If we can update the variable name to `envVars` or something like this it would be good.